### ISSUE TITLE
Page 107 g.setFont (40.0f)  is not working in Juce 8

### ISSUE THREAD

**Issue by mpaiato** [TheAudioProgrammer/getting-started-book#2]
In juce 8 it need to be
       g.setFont (juce::FontOptions (40.0f));

**Comment by hollance** [comment]
Yes, this has changed in JUCE 8. Writing `g.setFont(40.0f);` should still work, but it is deprecated, which means that it is no longer the preferred way of setting font sizes.
